packages/components/time-select/__tests__/time-select.test.tsx

```diff
@@ -74,7 +73,7 @@ describe('TimeSelect', () => {
     value.value = '10:30'
     await nextTick()
 
-    expect(wrapper.findComponent(TimeSelect).vm.value).toBe('10:30')
+    expect(value.value).toBe('10:30')
```

Comment:
IMO, the test looks redundant by this way.

Reply:
Thanks for the review.